CHANGELOG.md

```diff
@@ -5,12 +5,33 @@ Notable changes to this project are documented in this file. The format is based
 ## [Unreleased]
 
 Breaking changes:
+- Updated code for PureScript 0.14 (#181)
+- Renamed `Data.Array.ST.empty` to `Data.Array.ST.new` (#191 #198)
+- Renamed `group'` to `groupAll` (#194 #200)
 
 New features:
+- Added monomorphic `foldl` `foldr` `foldMap` `fold` `intercalate` to `Array` (#201)
+  - These are just wrapped versions from `Data.Foldable`
+- Added monomorphic `foldl1` `foldr1` `foldMap1` `foldl1` `intercalate` to `Array.NonEmpty` (#201)
+  - These are just wrapped versions from `Data.Foldable`
```

Comment:
I'm not sure "wrapped versions" is the correct terminology here; @hdgarrood, do you have a suggestion?

Reply:
Yeah, I think “wrapped” suggests that actually you’ve done something to them, whereas all we’ve done here is specialize. I’d suggest “specialized versions of the functions with the same names from Data.Foldable”. Is that true for all of these though? This shouldn’t be a separate bullet point either, it’s part of the same change.

Reply:
To be clear, the one I’m not sure about is intercalate - I thought it was supposed to get its own Array-specific implementation.

Reply:
> This shouldn’t be a separate bullet point either, it’s part of the same change.

It's indented, to indicate a clarification rather than an entire new point, but I think it reads a little better as just sentences in the same bullet point and agree it should just merge into the previous line.